snow/README.md

```diff
@@ -16,7 +16,7 @@ graph LR
 
 ## Intro
 
-The Avalanche network consists of 3 built-in blockchains: X-Chain, C-Chain, and P-Chain. The X-Chain is used to manage assets and uses the Avalanche consensus protocol. The C-Chain is used to create and interact with smart contracts and uses the Snowman consensus protocol. The P-Chain is used to coordinate validators and stake and also uses the Snowman consensus protocol. At the time of writing, the Avalanche network has ~1200 validators. A set of validators makes up a subnet. Subnets can validate 1 or more chains. It is a common misconception that 1 subnet = 1 chain and this is shown by the primary subnet of Avalanche which is made up of the X-Chain, C-Chain, and P-Chain.
+The Avalanche network consists of 3 built-in blockchains: X-Chain, C-Chain, and P-Chain. The X-Chain is used to manage assets and uses the Snowman consensus protocol. Previously the X-Chain used the Avalanche consensus protocol, which was DAG-based, but the consensus engine was updated to Snowman in avalanchego v1.10.1. The C-Chain is used to create and interact with smart contracts and uses the Snowman consensus protocol. The P-Chain is used to coordinate validators and stake and also uses the Snowman consensus protocol. At the time of writing, the Avalanche network has ~1200 validators. A set of validators makes up a subnet. Subnets can validate 1 or more chains. It is a common misconception that 1 subnet = 1 chain and this is shown by the primary subnet of Avalanche which is made up of the X-Chain, C-Chain, and P-Chain.
```

Comment:
> was updated to Snowman in avalanchego v1.10.1.

Should be: "was updated to Snowman in the Cortina network upgrade."

Also I feel like we could more susinctly say that all chains are now based on Snowman consensus.. Rather than explicitly saying that the P-chain and the C-chain use Snowman

Reply:
Sure, sounds good. I agree it's overly verbose as-is.